src/VisualStudio/Core/Def/Implementation/FindReferences/Contexts/AbstractTableDataSourceFindUsagesContext.cs

```diff
@@ -76,11 +76,16 @@ private abstract class AbstractTableDataSourceFindUsagesContext :
             private TableEntriesSnapshot _lastSnapshot;
             public int CurrentVersionNumber { get; protected set; }
 
+            /// <summary>
+            /// Column states for custom columns.
+            /// </summary>
+            private readonly Dictionary<string, ColumnState2> _customColumnStatesMap;
             #endregion
```

Comment:
nit: blank line above #endregion.

Reply:
just to be clear.  You want this in this region right?  i.e. this is intended ot be protected by _gate?  if so, great. but i just want to check.

Reply:
Yes, correct - it currently has all uses protected by _gate.

---
In reply to: [222056491](https://github.com/dotnet/roslyn/pull/30155#discussion_r222056491) [](ancestors = 222056491)